R/layer_yeo_johnson.R

```diff
@@ -0,0 +1,249 @@
+#' Unormalizing transformation
+#'
+#' Will undo a step_epi_YeoJohnson transformation.
+#'
+#' @param frosting a `frosting` postprocessor. The layer will be added to the
+#'   sequence of operations for this frosting.
+#' @param lambdas Internal. A data frame of lambda values to be used for
```

Comment:
1. This should likely not be labeled as internal. In principle, we'd like layers to operate outside a workflow. So allowing a user to pass the necessary object here should be allowed.

2. I don't like the name `lambdas`. Better to choose something non-jargon, but understandable. See e.g., https://www.tidymodels.org/learn/develop/models/#step-1.-register-the-model-modes-and-arguments

Reply:
`yj_parameters`?